plugins/dell-kestrel/fu-dell-kestrel-ec.c

```diff
@@ -585,8 +596,8 @@ fu_dell_kestrel_ec_get_package_version(FuDellKestrelEc *self)
 gboolean
 fu_dell_kestrel_ec_commit_package(FuDellKestrelEc *self, GInputStream *stream, GError **error)
 {
-	g_autoptr(GByteArray) req = g_byte_array_new();
-	g_autoptr(GBytes) bytes = NULL;
+	g_autoptr(GByteArray) req = fu_struct_dell_kestrel_ec_databytes_new();
+	g_autoptr(GByteArray) bytes = NULL;
```

Comment:
I think `buf` is the more normal variable name for a mutable `GByteArray` -- `bytes` implies a `GBytes`.